GPU/Common/DrawEngineCommon.cpp

```diff
@@ -645,7 +646,7 @@ void DrawEngineCommon::DecodeVertsStep(u8 *dest, int &i, int &decodedVerts) {
 		for (int j = i + 1; j < total; ++j) {
 			if (drawCalls_[j].verts != dc.verts)
 				break;
-
+			// TODO: What if UV scale/offset changes between drawcalls here?
```

Comment:
Hm, this probably is a problem.  I'm surprised we haven't noticed before, but maybe it's the cause of some weird graphics issues?

Maybe we can just set a flag when UVs change mid draw call (they often don't), and skip this optimization when it happens?

-[Unknown]